pinc/ProjectTransition.inc

```diff
@@ -1260,31 +1260,53 @@ new ProjectTransition(
 // Transitions to PROJ_SUBMIT_PG_POSTED:
 
 /*
-    pseudo-code:
+    The following table describes the required values for checkedoutby,
+    postproofer  and ppverifier after the transition to the named state.
+    The states are listed in their normal order, though projects may
```

Comment:
Incomplete sentence -- don't leave me hanging! 😁 

(also, double space mid-sentence in the line above)

Reply:
Done! I did complete that sentence once, so either changed my mind and decided to reword it, then forgot, or I messed up syncing the latest version of the file from my local system. Thanks for spotting it anyway.